markdown/chapters/entombsday/deck-construction.md

```diff
@@ -0,0 +1,52 @@
+---
+authors: drynne
+order: 2
+title: Deck Construction
+---
+
+## Table of Contents
+
+## Preamble
+
+As previously stated, the shell of both Tin Fins and Doomsday are remarkably similar and the combo packages are roughly the same size. Because of this overlap, there are very few concessions made in deckbuilding.
+
```

Comment:
Maybe introduce the two lists and give some discussion on them and the choices made for each / reasoning for any differences?

"Here are some example decklists". You can see the shell provides the basis of Tin Fins with xyz differences. etc etc